app/src/main/java/com/vultisig/wallet/ui/components/library/form/Form.kt

```diff
@@ -245,4 +266,45 @@ internal fun FormDetails(
             style = Theme.menlo.body1
         )
     }
+}
+
+
+@Composable
+internal fun TextFieldValidator(
+    errorText: UiText?,
+    focusState: FocusState?,
+    content: @Composable () -> Unit,
+) {
+    var errorMessage by remember {
+        mutableStateOf<UiText?>(null)
+    }
```

Comment:
I define `errorMessage` as a state because its value changes based on the focus state. If the text field is in focus state, the error message is null, so it will not be shown. However, if the text field loses its focus, the error message gets its value and is shown."
Actually, the value of `errorMessage` changes depending on the focus situation.